### PR TITLE
BracesFixer - do not remove empty lines after class opening

### LINKED ISSUE #1404: Braces fixer removes empty lines after class opening
Before:
```php
class simpleTestClass
{

    private $foo;

    private $bar;
}
```

After running PSR-2 level fixer "braces" fixes it to:
```php
class simpleTestClass
{
    private $foo;

    private $bar;
}
```

According to #567, #874, #890 - it was fixed and #885 made extra lines removal into a separate fixer. However I'm running CS v.1.10 and my code is still getting "fixed". Is there any reason why a default PSR2 fixer removes those lines?

### PR DESCRIPTION
**Pull request by keradus**
Fixes #1404 
Bug introduced by 84d9fb5

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
I guess we also need a fixer that will remove these extra lines?

**Comment by keradus** [comment]
perhaps on some other level

**Comment by GrahamCampbell** [comment]
Yeh, of course. ;)

File: Symfony/CS/Fixer/PSR2/BracesFixer.php [hunk 1]
```diff
@@ -268,7 +268,19 @@ function ($item) {
...
                 $nextToken = $tokens[$startBraceIndex + 1];
-                $tokens->ensureWhitespaceAtIndex($startBraceIndex + 1, 0, "\n".$indent.'    ');
+
+                if (!$nextToken->isWhitespace()) {
+                    $tokens->ensureWhitespaceAtIndex($startBraceIndex + 1, 0, "\n".$indent.'    ');
+                }
+                else {
+                    $tmpIndent = trim($nextToken->getContent(), " \t").$indent.'    ';
+
+                    if (!isset ($tmpIndent[0]) || "\n" !== $tmpIndent[0]) {
```
Comment (sstok) [inline Symfony/CS/Fixer/PSR2/BracesFixer.php]: Remove space between isset and `(`

File: Symfony/CS/Fixer/PSR2/BracesFixer.php [hunk 2]
```diff
@@ -268,7 +268,19 @@ function ($item) {
                 }
             } else {
                 $nextToken = $tokens[$startBraceIndex + 1];
-                $tokens->ensureWhitespaceAtIndex($startBraceIndex + 1, 0, "\n".$indent.'    ');
+
+                if (!$nextToken->isWhitespace()) {
+                    $tokens->ensureWhitespaceAtIndex($startBraceIndex + 1, 0, "\n".$indent.'    ');
+                }
+                else {
```
Comment (sstok) [inline Symfony/CS/Fixer/PSR2/BracesFixer.php]: Should be on the same line as the `}`

**Comment by keradus** [comment]
@fabpot 
PHP CS Fixer found CS issues, eg `braces` (from `PSR2` level) .
Why fabbot.io didn't notice that?

File: Symfony/CS/Fixer/PSR2/BracesFixer.php (hunk 1 above)
Comment (keradus) [inline Symfony/CS/Fixer/PSR2/BracesFixer.php]: thx

File: Symfony/CS/Fixer/PSR2/BracesFixer.php (hunk 2 above)
Comment (keradus) [inline Symfony/CS/Fixer/PSR2/BracesFixer.php]: thx

**Pull request closed without merging**